### PR TITLE
Increase memory limit and add make target to update bundle

### PR DESCRIPTION
**Pull request by dagrayvid**

### CODE REVIEW

File: Makefile.helper.mk [hunk 1]
```diff
@@ -62,3 +62,15 @@ else
...
+	mv $$(find bundle -name image-references) bundle/image-references
+	rm -rf bundle/4.*/manifests bundle/4.*/metadata
+	$(MAKE) bundle TAG=$(TAG) VERSION=$(VERSION) IMAGE=$(IMAGE)
+	mv bundle/manifests/special-resource-operator.clusterserviceversion.yaml bundle/manifests/special-resource-operator.v$(VERSION).clusterserviceversion.yaml
+	mv bundle/manifests bundle/$(TAG)/manifests
+	mv bundle/metadata bundle/$(TAG)/metadata
+	sed 's#bundle/##g' bundle.Dockerfile | head -n -1 > bundle/$(TAG)/bundle.Dockerfile
+	mv bundle/image-references bundle/$(TAG)/manifests/image-references
+
+
```
Comment (jmencak) [inline Makefile.helper.mk]: Why that many new lines?

**Review by jmencak — COMMENTED** [review]
Bumping the memory limits/requests makes sense, cannot comment on the bundle target.

File: Makefile.helper.mk (hunk 1 above)
Comment (dagrayvid) [reply Makefile.helper.mk]: Fixed, thanks

**Pull request merged**